src/ai_hawk/job_manager.py

```diff
@@ -239,7 +241,7 @@ def start_applying(self):
                     time.sleep(sleep_time)
                 page_sleep += 1
 
-    def get_jobs_from_page(self):
+    def get_jobs_from_page(self, scroll=False):
```

Comment:
I'm curious, why do you have the default value as False?

Reply:
to use it only when required.